### DISCUSSION TITLE
auto-reload option

### DISCUSSION THREAD

**Discussion by 1925381584** [helix-editor/helix#11316] [Ideas]
hi, I think the option of auto-reload to let the buffer auto revert is useful.
when it open, the buffer changed outside will be sync into inside auto, if the buffer changed inside also, Give a hint whether to force an override. does it has exist?

**Comment by the-mikedavis** [comment]
See https://github.com/helix-editor/helix/issues/1125, we need a file system monitor (which @pascalkuthe is working on as time allows) to be able to pick up on external changes
